wezterm-gui/src/overlay/copy.rs

```diff
@@ -66,6 +67,7 @@ struct CopyRenderable {
 
     /// The text that the user entered
     pattern: Pattern,
+    search_cursor: StableCursorPosition,
```

Comment:
FWIW, I would just use `usize` for this, as `StableCursorPosition` refers to a location in scrollback while the `search_cursor` should just be a column within the search field.

Reply:
Fixed